cmd/security-proxy-setup/res/docker/configuration.toml

```diff
@@ -66,37 +66,63 @@ SNIS = ["edgex-kong"]
   Host = "edgex-core-command"
   Port = 48082
 
-  [Clients.Notifications]
+[Services]
+  [Services.Coredata]
+  Name = "Coredata"
   Protocol = "http"
-  Host = "edgex-support-notifications"
+  Host = "localhost"
```

Comment:
this seems wrong for the docker configuration.toml file, shouldn't it be the name of the docker container to use as the hostname?

Reply:
I called this out separately via Slack. He's on it.

Reply:
@anonymouse64 , @tsconn23 - Ty for the callout...fixed and updated the PR.